roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -1,107 +1,453 @@
-# Statement Distribution
-
-The Statement Distribution Subsystem is responsible for distributing statements about seconded candidates between validators.
+# Statement Distribution (Draft)
+
+This is a first draft for the implementer's guide, which I wrote as a dump of
+all info and conclusions I could find, but in one place. This helped me a lot to
+connect all the "dots", but it may be too much info. I need feedback on the
+following:
+
+-   [ ] Where it is too technical, I can move stuff into mod-docs.
+-   [ ] And of course, accuracy/thoroughness.
+
+There is some redundancy, but I came across a saying I liked today: "DRY is for
+code, not docs." Re-stating the same thing multiple times helped with my
+comprehension. But feedback is welcome.
+
+Also, I will convert it to prose once we know what info is staying. The bullets just helped me stay organized.
+
+## Overview
+
+-   ****Goal:**** every well-connected node is aware of every next potential parachain
+    block.
+-   Validators can either:
+    -   receive parachain block from collator, check block, and gossip statement.
+    -   receive statements from other validators, check block, gossip forward
+        statement if valid.
+-   Why:
+    -   Validators must have statements from all (?) other validators
+    -   Why:
+        -   Quorum must be reached before the block author will consider the
+            candidate.
+        -   We need to store statements from validators who've checked the candidate
+            on the relay chain, so we know who to hold accountable in case of
+            disputes.
+    -   "Validators who aren't assigned to the parachain still listen for the
+        attestations because whichever validator ends up being the author of the
+        relay-chain block needs to bundle up attested parachain blocks for several
+        parachains and place them into the relay-chain block."
+
+### With Async Backing
+
+Async backing changes it so that:
+
+-   We allow inclusion of old parachain candidates validated by current
+    validators.
+-   We allow inclusion of old parachain candidates validated by old validators.
```

Comment:
This section isn't exactly clear or maybe not correct.

Maybe better rephrased: "asynchronous backing changes the runtime to accept parachain candidates from a certain allowed range of historic relay-parents. these candidates must be backed by the group assigned to the parachain as-of their corresponding relay parent".